docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1896,7 +1912,9 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0072W":"Could not stop manager, error=
+### ZWED0072W
+
+  Could not stop manager, error=
 
```

Comment:
Do we need error=**%s** ?

Reply:
i havent touched these yet, i was just bulk fixing formatting.